packages/editor-ui/src/components/canvas/Canvas.vue

```diff
@@ -562,7 +564,12 @@ provide(CanvasKey, {
 
 		<CanvasArrowHeadMarker :id="arrowHeadMarkerId" />
 
-		<Background data-test-id="canvas-background" pattern-color="#aaa" :gap="GRID_SIZE" />
+		<Background data-test-id="canvas-background" pattern-color="#aaa" :gap="GRID_SIZE">
+			<template v-if="readOnly" #pattern-container>
+				<CanvasBackgroundStripedPattern :x="viewport.x" :y="viewport.y" :zoom="viewport.zoom" />
+				<rect x="0" y="0" width="100%" height="100%" fill="url(#diagonalHatch)" />
```

Comment:
Could we move the `<rect>` inside of `CanvasBackgroundStripedPattern`? We should be able to render multiple roots like a Fragment without problems.

Reply:
it should work but eslint complains
![image](https://github.com/user-attachments/assets/f2e3afa0-ff55-46ff-a58b-1ece9a44ea48)

Reply:
done